clase1/enunciado1/src/main/java/Curso.java

```diff
@@ -0,0 +1,94 @@
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+
+@Getter
+@Setter
+public class Curso {
+    private String nombre;
+    private int cantHoras;
+    private ArrayList<Alumno> alumnos;
+
+    public Curso(String nombre, int cantHoras, ArrayList<Alumno> alumnos) {
+        this.nombre = nombre;
+        this.cantHoras = cantHoras;
+        this.alumnos = alumnos;
+    }
+
+    public void setAlumnos(Alumno a) {
```

Comment:
no hace falta porque tenes el @Setter

Reply:
claro pero el setter que me crea por defecto es el siguiente:
public void setAlumnos(ArrayList<Alumno> alumnos) {
        this.alumnos = alumnos;
    }
por lo tanto es incorrecto y me tira error cuando le quiero agregar un objeto alumno al array alumnos.